modules/cluster-autoscaler-cr.adoc

```diff
@@ -41,10 +41,10 @@ spec:
 ----
 <1> Specify the priority that a pod must exceed to cause the cluster autoscaler to deploy additional nodes. Enter a 32-bit integer value. The `podPriorityThreshold` value is compared to the value of the `PriorityClass` that you assign to each pod.
 <2> Specify the maximum number of nodes to deploy. This value is the total number of machines that are deployed in your cluster, not just the ones that the autoscaler controls. Ensure that this value is large enough to account for all of your control plane and compute machines and the total number of replicas that you specify in your `MachineAutoscaler` resources.
-<3> Specify the minimum number of cores to deploy in the cluster.
-<4> Specify the maximum number of cores to deploy in the cluster.
-<5> Specify the minimum amount of memory, in GiB, in the cluster.
-<6> Specify the maximum amount of memory, in GiB, in the cluster.
+<3> Specify the minimum number of cores to deploy on the worker and control plane nodes in the cluster.
+<4> Specify the maximum number of cores to deploy on the worker and control plane nodes in the cluster.
+<5> Specify the minimum amount of memory, in GiB, on the worker and control plane nodes in the cluster.
+<6> Specify the maximum amount of memory, in GiB, on the worker and control plane nodes in the cluster.
```

Comment:
the only question i have here is will this be confusing to a reader who sees the "control plane nodes" part of this, the autoscaler does not manage the control plane nodes even though it takes them into account when calculating these values.